File: recipesharp/Models/Recipe.cs
namespace recipesharp.Models;

public class Recipe
{
    public int Id { get; set; } 
    public string  Title { get; set; }  
    public string Instructions { get; set; }  
    public string ImgUrl { get; set; }  
    public string Category { get; set; }  
    public string CreatorId { get; set; } 
    public Account Creator { get; set; }
}

public class RecipeLiker : Recipe
{
    public int RecipeMemberId { get; set; } 
}

File: recipesharp.client/src/models/Recipe.js

export class Recipe{

  constructor(data){
    this.id = data.id
    this.title = data.title
    this.instructions = data.instructions
    this.imgUrl = data.imgUrl
    this.category = data.category
    this.creatorId = data.creatorId
    this.creator = data.creator
  }
}